### ISSUE TITLE
Update `node-buffer` to `v9.0.0`

### ISSUE THREAD

**Issue by JordanMartinez** [purescript-node/purescript-node-buffer-blob#2]
I'm making my rounds updating the Node libraries. This may need to be updated for this library to be useable once the Node libraries' breaking changes hit the package set.

https://github.com/purescript-node/purescript-node-buffer-blob/blob/a2c6109fd5d7f73bd84b7827d4e31e959c61f009/bower.json#L25

**Comment by srghma** [comment]
yes, the package is not in registry, latest time it was in package-set is 3something

repo:purescript/registry node-buffer-blob path:/^package-sets\/3\d.*/
